ci/quality-base.sh

```diff
@@ -1,55 +1,54 @@
 #!/bin/bash
 
-source pipx-base.sh
+source uvx-base.sh
 
 python_files_and_folders() {
-    echo $(python $(script_dir)/python_files_and_folders.py)
+    echo $(.venv/bin/python $(script_dir)/python_files_and_folders.py)
 }
 
 run_ruff() {
     PYTHON_FILES_AND_FOLDERS=$(python_files_and_folders)
-    run_pipx ruff check $PYTHON_FILES_AND_FOLDERS
-    run_pipx ruff format --check $PYTHON_FILES_AND_FOLDERS
+    run_uvx ruff check $PYTHON_FILES_AND_FOLDERS
+    run_uvx ruff format --check $PYTHON_FILES_AND_FOLDERS
 }
 
 run_fixit() {
-    run_pipx fixit lint $(python_files_and_folders)
+    run_uvx fixit lint $(python_files_and_folders)
 }
 
 run_mypy() {
     PYTHON_FILES_AND_FOLDERS=$(python_files_and_folders)
     # Run mypy with or without pydantic plugin depending on whether pydantic is listed as dependency in the tools
     # section of the optional dependencies in the pyproject.toml file.
-    pydantic_spec=$(spec pydantic)
+    pydantic_spec=$(spec pydantic ==)
```

Comment:
This seems to be the only place where spec is called with a `==` separator, why the distinction?

Reply:
See above.